src/app/pages/data-protection/scrub/resilver-config/resilver-config.component.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit,
+} from '@angular/core';
+import { Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { of } from 'rxjs';
+import helptext from 'app/helptext/storage/resilver/resilver';
+import { FormErrorHandlerService } from 'app/pages/common/ix-forms/services/form-error-handler.service';
+import { DialogService, TaskService, WebSocketService } from 'app/services';
+import { CalendarService } from '../../../../services/calendar.service';
+import { EntityUtils } from '../../../common/entity/utils';
+
+@UntilDestroy()
+@Component({
+  templateUrl: './resilver-config.component.html',
+  styleUrls: ['./resilver-config.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ResilverConfigComponent implements OnInit {
+  isFormLoading = false;
+
+  form = this.fb.group({
+    enabled: [true],
+    begin: [''],
+    end: [''],
+    weekday: [[1, 2, 3, 4, 5, 6, 7], Validators.required],
```

Comment:
Set the default value with Weekday enum like `weekday: [[Weekday.Mon, Weekday.Tue, Weekday.Wed....] as Weekday[], ...`